src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,82 @@
+import { useCallback, useMemo, useState } from 'react';
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+
+interface Props {
+  people: Person[];
+}
+
+export const PeopleTable = ({ people }: Props) => {
```

Comment:
```suggestion
export const PeopleTable: React.FC<Props> = ({ people }: Props) => {
```
why no type? 😢

Reply:
cz in new version react it didn't necessary

Reply:
it is like writing const age: number = 35; I can but for what?